development/tools/pkg/githubrelease/release.go

```diff
@@ -0,0 +1,75 @@
+package githubrelease
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+// Release .
+type Release struct {
+	Gap *GithubAPIWrapper
```

Comment:
Gap and Saw are not the best names IMO. Maybe we can rename it to Github and Storage?